request-controller/src/main/java/org/wildfly/extension/requestcontroller/RequestController.java

```diff
@@ -59,7 +59,7 @@
  */
 public class RequestController implements Service<RequestController>, ServerActivity {
 
-    public static final ServiceName SERVICE_NAME = ServiceName.JBOSS.append("server", "global-request-controller");
+    public static final ServiceName SERVICE_NAME = RequestControllerRootDefinition.REQUEST_CONTROLLER_CAPABILITY.getCapabilityServiceName();
```

Comment:
Just to add to Brian is the real authority here. I'm just suggesting this based on what was done elsewhere. If this can wait until after Beta1 when he's back that's fine too. I don't know that this is wrong, it's just not what we've done else where that I know of.